src/Cynthia.Card/src/Cynthia.Card.Server/Services/GwentGameService/GwentServerService.cs

```diff
@@ -736,9 +707,12 @@ public async void MMRTrinkets(string PlayerName, int mymmr) // add trinkets when
                         ranktitle = "GRANDMASTER";
                         break;
                 }
-                await AddBorder(PlayerName, rank);
-                await AddTitle(PlayerName, ranktitle);
-                await AddAvatar(PlayerName, rankavatar);
+                var connectionId = _users.Single(x => x.Value.PlayerName == PlayerName).Value.ConnectionId;
```

Comment:
if the player is not online when calling this function, the server will crash. please chekc the result is empty before calling `Single `